### DISCUSSION TITLE
Account suspended and support ignores me

### DISCUSSION THREAD

**Discussion by oskarssarns** [community/community#55444] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My friend was creating a project on May 8th, when he suddenly got the message "Account suspended. Access to your account has been suspended due to a violation of our Terms of Service.".

He sent 4 requests, but without an answer, here is tickets:
#2146332
#2148455
#2150350
#2153829

I would like to know if you guys can move the appeal or do something about it; his account is https://github.com/BrunoGoretti

Thank you

**Comment by samruddhikhandale** [comment]
Hi 👋 

Thanks for opening a discussion and apologies for the inconvenience.
 As this issue is not (directly) related to Codespaces, I have sent a request to forward this discussion to the designated team.

**Comment by ettaboyle** [comment, marked as answer]
Hi @sharns Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. 

Thank you!
